classes/local/util/manager.php

```diff
@@ -264,6 +268,28 @@ protected function prepare_observers() {
         }
     }
 
+    /*
+     * Prepare the event class files.
+     */
+    protected function prepare_events() {
+
+        foreach ($this->recipe['events'] as $event) {
+            if (empty($event['eventname'])) {
+                throw new exception('Missing event name');
+            }
+
+            $eventrecipe = $this->recipe;
+            $eventrecipe['event'] = $event;
```

Comment:
I was thinking the same thing, that we should use a dedicated area for these things, to avoid name collisions in the future. I'll try to implement this, see how it looks.